crates/ironclaw_gateway/static/index.html

```
@@ -527,9 +527,15 @@ <h3 data-i18n="skills.installed">Installed Skills</h3>
               </div>
               <div class="extensions-section">
                 <h3 data-i18n="skills.installByUrl">Install Skill by URL</h3>
-                <div class="ext-install-form">
-                  <input type="text" id="skill-install-name" data-i18n-placeholder="skills.namePlaceholder" placeholder="Skill name or slug">
-                  <input type="text" id="skill-install-url" data-i18n-placeholder="skills.urlPlaceholder" placeholder="HTTPS URL to SKILL.md (optional)">
+                <div class="ext-install-form skill-install-form" id="skill-install-form">
+                  <div class="skill-install-field">
+                    <input type="text" id="skill-install-name" data-i18n-placeholder="skills.namePlaceholder" placeholder="Skill name or slug">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)